internal/persistence/queries_test.go

```diff
@@ -0,0 +1,350 @@
+		require.NoError(t, err, "failed to fetch task log")
+
+		taskAfter, err := fetchTaskByID(testDB, taskID)
+		require.NoError(t, err, "failed to fetch task")
+
+		assert.Equal(t, numSeconds, taskLog.SecsSpent)
+		assert.Equal(t, comment, taskLog.Comment)
+		assert.Equal(t, numSecondsBefore+numSeconds, taskAfter.SecsSpent)
+	})
+
+	t.Run("TestInsertManualTL", func(t *testing.T) {
+		t.Cleanup(func() { cleanupDB(t, testDB) })
+
+		// GIVEN
+		referenceTS := time.Now()
+		seedData := getTestData(referenceTS)
+		seedDB(t, testDB, seedData)
+		taskID := 1
+
+		taskBefore, err := fetchTaskByID(testDB, taskID)
+		require.NoError(t, err, "failed to fetch task")
+		numSecondsBefore := taskBefore.SecsSpent
+
+		// WHEN
+		comment := "a task log"
+		numSeconds := 60 * 90
+		endTS := time.Now()
+		beginTS := endTS.Add(time.Second * -1 * time.Duration(numSeconds))
+		tlID, err := InsertManualTL(testDB, taskID, beginTS, endTS, comment)
+
+		// THEN
+		require.NoError(t, err, "failed to insert task log")
+
+		taskLog, err := fetchTaskLogByID(testDB, int(tlID))
+		require.NoError(t, err, "failed to fetch task log")
+
+		taskAfter, err := fetchTaskByID(testDB, taskID)
+		require.NoError(t, err, "failed to fetch task")
+
+		assert.Equal(t, numSeconds, taskLog.SecsSpent)
+		assert.Equal(t, comment, taskLog.Comment)
+		assert.Equal(t, numSecondsBefore+numSeconds, taskAfter.SecsSpent)
+	})
+
+	t.Run("TestDeleteTaskLogEntry", func(t *testing.T) {
+		t.Cleanup(func() { cleanupDB(t, testDB) })
+
+		// GIVEN
+		referenceTS := time.Now()
+		seedData := getTestData(referenceTS)
+		seedDB(t, testDB, seedData)
+		taskID := 1
+		tlID := 1
+		taskBefore, err := fetchTaskByID(testDB, taskID)
+		require.NoError(t, err, "failed to fetch task")
+		numSecondsBefore := taskBefore.SecsSpent
+		taskLog, err := fetchTaskLogByID(testDB, tlID)
+		require.NoError(t, err, "failed to fetch task log")
+
+		// WHEN
+		err = DeleteTaskLogEntry(testDB, &taskLog)
+
+		// THEN
+		require.NoError(t, err, "failed to insert task log")
+
+		taskAfter, err := fetchTaskByID(testDB, taskID)
+		require.NoError(t, err, "failed to fetch task")
+
+		assert.Equal(t, numSecondsBefore-taskLog.SecsSpent, taskAfter.SecsSpent)
+	})
+
+	t.Run("TestFetchStats", func(t *testing.T) {
+		t.Cleanup(func() { cleanupDB(t, testDB) })
+
+		// GIVEN
+		referenceTS := time.Date(2024, time.September, 1, 9, 0, 0, 0, time.Local)
+		seedData := getTestData(referenceTS)
+		seedDB(t, testDB, seedData)
+
+		taskID := 1
+		comment := "an extra task log"
+		numSeconds := 60 * 90
+		tlEndTS := referenceTS.Add(time.Hour * 2)
+		tlBeginTS := tlEndTS.Add(time.Second * -1 * time.Duration(numSeconds))
+		_, err = InsertManualTL(testDB, taskID, tlBeginTS, tlEndTS, comment)
+		require.NoError(t, err, "failed to insert task log")
+
+		// WHEN
+		entries, err := FetchStats(testDB, 100)
+
+		// THEN
+		require.NoError(t, err, "failed to fetch report entries")
+
+		require.Equal(t, 2, len(entries))
```

Comment:
```suggestion
		require.Len(t, entries, 2)
```

I would like to suggest you adding "testifylint" linter in golangci-lint. It detects this